tests/gateway/test_runner.py

```diff
@@ -205,33 +208,54 @@ def test_server_update_with_invalid_config(
     config = str(store_conf(tmp_path, "config.yaml", basic_config_dict))
 
     with Gateway(config) as gateway:
-        response = gateway.request("gateway/routes/")
+        response = gateway.get("gateway/routes/")
         assert response.json() == basic_routes
         # Give filewatch a moment to cycle
         wait()
```

Comment:
`wait()` here is useless because we don't update the config after the gateway starts. I'll remove this in a follow-up.